src/applications/appeals/995/tests/utils/submit.unit.spec.js

```diff
@@ -3,6 +3,7 @@ import {
   PRIMARY_PHONE,
   SELECTED,
   EVIDENCE_VA,
+  EVIDENCE_OTHER,
```

Comment:
Did you mean to remove `EVIDENCE_OTHER`, I see the code still has it on line 299: 
`[EVIDENCE_OTHER]: true,`